src/Formation.php

```diff
@@ -602,13 +610,25 @@ public function import(): array
         return [];
     }
 
+    public function export(): array
+    {
+        return $this->index();
```

Comment:
Actually lets return an empty array here.. that way if we auto-generate routes like we do [here](https://github.com/headlesslaravel/headlesslaravel/blob/main/src/Manager.php#L85-L90) 

We will know that they want an export for that formation if they defined fields
```suggestion
        return [];
```

Reply:
Done